tests/unit/test_package.py

```diff
@@ -51,39 +51,22 @@ def test_rpm_install_nodeps(self, mock_sudo):
         mock_sudo.assert_called_with('rpm -i --nodeps '
                                      '/opt/prestoadmin/packages/test.rpm')
 
+    @patch('prestoadmin.package._rpm_uninstall')
+    @patch('prestoadmin.package._rpm_install')
     @patch('prestoadmin.package.sudo')
-    def test_rpm_upgrade(self, mock_sudo):
+    def test_rpm_upgrade(self, mock_sudo, mock_rpm_install, mock_rpm_uninstal):
```

Comment:
it was by purpose, `l` caused flake to fail and I didn't want to break this line ;) Now, once I line length limit is extended I will fix that ;)